apps/avifenc.c

```diff
@@ -1213,8 +1213,8 @@ static avifBool avifEncodeImagesFixedQuality(const avifSettings * settings,
            encoder->autoTiling ? "automatic tiling" : manualTilingStr,
            settings->jobs);
     if (settings->progressive) {
-        // If the color quality is less than 10, the main() function overrides
-        // --progressive and sets settings->autoProgressive to false.
+        // If --progressive is specified and the color quality is less than 10,
```

Comment:
```diff
-// If --progressive is specified and the color quality is less than 10,
+// If --progressive is specified and the color quality is less than PROGRESSIVE_WORST_QUALITY,
```

to avoid the comment going out of sync in the future

Reply:
Done.